x-pack/plugins/apm/scripts/bootstrap_apm/README.md

```diff
@@ -0,0 +1,56 @@
+# Getting started
+
+A script for creating an elastic stack deployment on a remote cluster using `ecctl` and generating APM data using apm-sythtrace tool.
+
+## Prerequisites for running the script
+
+- [ecctl](https://www.elastic.co/downloads/ecctl) installed
+- API key
+  - You can generate a key on [staging](https://staging.found.no/deployment-features/keys)
+- Make sure you have installed Kibana dependencies
+
+## Configuration
+
+Update the following configuration with your own credentials and store it your home directory `$HOME/.ecctl/config.json`
```

Comment:
I guess just using `ecctl init` won't give you https://staging.found.no as a default host choice, right?

Reply:
right, with `ecctl init` you can select between the 3 Elastic cloud offerings

1. Elasticsearch Service (default).
2. Elastic Cloud Enterprise (ECE).
3. Elasticsearch Service Private (ESSP).

For the option 1  (which is also the default) the host will be `https://cloud.elastic.co` for the other 2 you need to define URL of your service